sources/AutoFilerTab.cpp

```diff
@@ -216,7 +223,9 @@ AutoFilerTab::MessageReceived(BMessage* msg)
 		}
 		case MSG_SHOW_ADD_PANEL:
 		{
-			fFilePanel->Show();
+			fAddPanel->Window()->SetTitle("AutoFiler: Add folders");
+			fAddPanel->SetButtonLabel(B_DEFAULT_BUTTON, "Add");
```

Comment:
The SetTitle() and SetButtonLabel() could be done once right after creating the file panels, right?